docs/samples/Microsoft.ML.Samples/Trainers.cs

```diff
@@ -74,5 +76,122 @@ public static void SdcaRegression()
             Console.WriteLine($"RMS - {metrics.Rms}");  // 4.924493
             Console.WriteLine($"RSquared - {metrics.RSquared}");  // 0.565467
         }
+
+        public static void FastTreeRegression()
+        {
+            // Downloading a regression dataset from github.com/dotnet/machinelearning
+            // this will create a housing.txt file in the filsystem this code will run
+            // you can open the file to see the data. 
+            string dataFile = SamplesUtils.DatasetUtils.DownloadHousingRegressionDataset();
+
+            // Creating the ML.Net IHostEnvironment object, needed for the pipeline
+            var env = new LocalEnvironment(seed: 0);
+
+            // Creating the ML context, based on the task performed.
+            var regressionContext = new RegressionContext(env);
+
+            // Creating a data reader, based on the format of the data
+            var reader = TextLoader.CreateReader(env, c => (
+                        label: c.LoadFloat(0),
+                        features: c.LoadFloat(1, 6)
+                    ),
+                separator: '\t', hasHeader: true);
+
+            // Read the data, and leave 10% out, so we can use them for testing
+            var data = reader.Read(new MultiFileSource(dataFile));
+            var (trainData, testData) = regressionContext.TrainTestSplit(data, testFraction: 0.1);
```

Comment:
In this file, `TrainTestSplit`  is used in all of the examples, we may also want to showcase `CrossValidate`. Using `CrossValidate` gives the user an estimate of the variance in the models created. ([more info](https://github.com/dotnet/machinelearning/blob/9157ceacffe0106c01580d6b7aa79ce918048434/docs/code/MlNetCookBook.md#how-do-i-train-using-cross-validation))

One further comment: a user wants to use `TrainTestSplit` or `CrossValidate` to get an estimate of the model quality, but they also want to run a final run to create a model using 100% of their data. This final model is the one they want to ship to production. 
Hence our examples should follow the form: (1) TrainTest/CV, (2) A note to user to iterate on the model until happy, (3) Full re-train on 100% data, (4) ship to production.